### PR TITLE
otb-tracker: Dynamic timeout

### PR DESCRIPTION
**Pull request by Tikiwinkie**

### CODE REVIEW

**Comment by gregdel** [comment]
PKG_VERSION should be changed :)

File: otb-tracker/bin [hunk 1]
```diff
@@ -16,32 +16,82 @@ export OTB_TRACKER_INTERFACE
...
+	network_get_gateway OTB_TRACKER_DEVICE_GATEWAY "$OTB_TRACKER_INTERFACE"
 	config_load network
 	config_get OTB_TRACKER_DEVICE "$OTB_TRACKER_INTERFACE" ifname
-	if [ -n "$OTB_TRACKER_DEVICE" ] && [ -n "$OTB_TRACKER_INTERFACE_IP" ] && [ -n "$OTB_TRACKER_INTERFACE_GATEWAY" ];then
-		# execute specific tracker
-		OTB_TRACKER_RESULT=$(/usr/share/otb/trackers.d/"$OTB_TRACKER_CALL" "$@")
-		[ "$?" = 0 ] && OTB_TRACKER_STATUS="OK"
+
+	# execute specific tracker
+	if [ -n "$OTB_TRACKER_DEVICE" ] && [ -n "$OTB_TRACKER_DEVICE_IP" ] && [ -n "$OTB_TRACKER_DEVICE_GATEWAY" ];then
```
Comment (angt) [inline otb-tracker/bin]: we should just skip bad device no ?

Comment (angt) [inline otb-tracker/bin]: you should just sleep and continue if the interface is not viable for tracking
for exemple we shouldn't call post-tracking methods.

Comment (Tikiwinkie) [inline otb-tracker/bin]: I don't think so, because if you delete the gateway for example, you want the interface to be autorouted

**Pull request closed without merging**